src/outpack/index.py

```diff
@@ -19,26 +19,29 @@ def new():
 class Index:
     def __init__(self, path):
         self._path = pathlib.Path(path)
-        self.data = IndexData.new()
+        self._data = IndexData.new()
```

Comment:
Bit of churn to make this protected now I've added a separate method to get the refreshed data